cmake/remillConfig.cmake.in

```diff
@@ -14,46 +14,20 @@
 
 @PACKAGE_INIT@
 
-if(NOT TARGET sleigh::support)
-  find_package(sleigh CONFIG REQUIRED COMPONENTS Support Specs)
-endif()
-
-foreach(_comp ${remill_FIND_COMPONENTS})
-  if (NOT "VCPKG_DEPS" STREQUAL _comp)
-    set(remill_FOUND False)
-    set(remill_NOT_FOUND_MESSAGE "Unsupported component: ${_comp}. Only VCPKG_DEPS")
-  endif()
-
-  if ("VCPKG_DEPS" STREQUAL _comp)
-    set(remill_setup_vcpkg True)
-    if(NOT "x@VCPKG_ROOT@x" STREQUAL "xx" AND NOT USE_SYSTEM_DEPENDENCIES)
-      if (EXISTS "@VCPKG_ROOT@")
-        set(VCPKG_ROOT "@VCPKG_ROOT@"
-          CACHE PATH "Location of dependency libraries"
-          )
-        include(${CMAKE_CURRENT_LIST_DIR}/vcpkg_helper.cmake)
-        message(STATUS "Found VCPKG_ROOT: ${VCPKG_ROOT}")
-        set(vcpkgdeps_FOUND)
-      endif()
-    endif()
-  endif()
-endforeach()
+set(LLVM_MAJOR_VERSION @LLVM_MAJOR_VERSION@)
+set(LLVM_MINOR_VERSION @LLVM_MINOR_VERSION@)
```

Comment:
Why do we set this? Does the LLVM CMake module not already provide this information?

Reply:
I think maybe what should be done is something like:
`set(REMILL_VERSION @REMILL_VERSION@)`

This came up in the discussion of #630.

Reply:
Yeah, that looks better

Reply:
```suggestion
set(REMILL_LLVM_VERSION @REMILL_LLVM_VERSION@)
```